go.mod

```diff
@@ -0,0 +1,10 @@
+module go-pvaccess
```

Comment:
Why are you calling the module go-pvaccess and not github.com/quentinmit/go-pvaccess? I think that is forcing all the import path changes you made, and it means this package will no longer work without modules.